src/coreclr/vm/eeconfig.cpp

```diff
@@ -943,6 +942,52 @@ TypeNamesList::TypeNamesList()
     LIMITED_METHOD_CONTRACT;
 }
 
+namespace
+{
+    //
+    // Slight modification of Rob Pike's minimal regex from The Practice of Programming.
+    // https://en.wikipedia.org/wiki/The_Practice_of_Programming
```

Comment:
Does the book mention a license for code fragments?

Reply:
I haven't found anything about license in the book itself - written in 1999. The official site does have the following page which I can reference.

https://www.cs.princeton.edu/~bwk/tpop.webpage/code.html